lnbits/decorators.py

```diff
@@ -173,7 +173,7 @@ async def get_key_type(
                 pass
             else:
                 raise
-        except:
+        except Exception:
             raise
```

Comment:
- isn't this `except` redundant?

Reply:
Bare except will catch exceptions you almost certainly don't want to catch, including KeyboardInterrupt (the user hitting Ctrl+C) and Python-raised errors like SystemExit

If you don't have a specific exception you're expecting, at least except Exception, which is the base type for all "Regular" exceptions.

https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except

Reply:
looking at it the whole thing is suspicious. i would defer it to another PR